doc/sphinx/source/modules/modules.rst

```diff
@@ -6,6 +6,10 @@ Standard Modules
 Standard modules are those which describe features that are considered
 part of the Chapel Standard Library.
 
+The modules :chpl:mod:`Assert` , :chpl:mod:`Math` , and :chpl:mod:`Types`
+are included by default.
```

Comment:
I considered this, but didn't like how "use'd" looks. This can be changed though.